frontend/src/scenes/instance/SystemStatus/RenderMetricValue.tsx

```diff
@@ -3,20 +3,34 @@ import { humanFriendlyDetailedTime } from 'lib/utils'
 import React from 'react'
 import { InstanceSetting } from '~/types'
 import { MetricRow } from './systemStatusLogic'
+import { LockOutlined } from '@ant-design/icons'
 
 const TIMESTAMP_VALUES = new Set(['last_event_ingested_timestamp'])
 
 type BaseValueInterface = Pick<MetricRow, 'key' | 'value'> & Partial<Pick<InstanceSetting, 'value_type'>>
 export interface MetricValueInterface extends BaseValueInterface {
     emptyNullLabel?: string
+    is_secret?: boolean
```

Comment:
Let's use camelCase in JS props (even if the Python API returns snake_case, this is simple to translate).